public/video-ui/src/components/FormFields/TagPicker.js

```diff
@@ -330,6 +331,17 @@ export default class TagPicker extends React.Component {
     }
   }
 
+  renderCharCount() {
+    if (this.props.tagType === TagTypes.youtube) {
+      const charCount = getYouTubeTagCharCount(this.props.fieldValue);
+      return (
+        <span>
+          Character count: {charCount}
```

Comment:
Could you put `{charCount} / {YouTubeKeywords.maxCharacters}` so it's clearer what the limit is as the user is typing?